src/main/webapp/college-info.html

```diff
@@ -6,33 +6,51 @@
     <link rel="stylesheet" href="style.css">
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
   </head>
-  <body>
+  <body onload="loadSchoolInfo()">
     <header>
       <a href="index.html"><h1>Black Student Insight</h1></a>
+      <h1 id="school-name"></h1>
       <input type="text" id="school-search" placeholder="Find your institution">
-      <button id="search-button">Search</button>
+      <button id="search-button" onclick="storeAndRedirect(); loadSchoolInfo();">Search</button>
     </header>
+
     <main>
-      <nav class="topnav">
-        <a class="active" href="college-info.html">School Info</a>
-        <a id="reviews-button">Student Reviews</a>
-      </nav>
-      <h2 id="school-name"></h2>
+      <div class="topnav">
+        <a id="info-button" class="active">School Info</a>
+        <a id="reviews-button" >Student Reviews</a>
+      </div>
+      <div id="main-info">
+        <section>
+          <p id="school-desc"></p>
+          <p id="students"></p>
+        </section>
 
-      <section>
-        <p id="school-desc"></p>
-        <p id="students"></p>
-      </section>
+        <section>
+          <p id="admissions"></p>
+          <p id="cost"></p>
+        </section>
 
-      <section>
-        <p id="admissions"></p>
-        <p id="cost"></p>
-      </section>
+        <section>
+          <p id="race-piechart"></p>
+          <p id="gender-piechart"></p>
+        </section>
+      </div>
 
-      <section>
-        <p id="race-piechart"></p>
-        <p id="gender-piechart"></p>
-      </section>
+      <div id="comment-body">
+        <h2>Student Reviews</h2>
+        <div id="comments-container"></div>
+          <h2>Leave a Review</h2>
+          <form id="submit-review" method="POST">
+            <input type="hidden" name="school-id" id="school-id">
+            <input name="name-input" placeholder="Name...">
+            <textarea name="text-input" placeholder="Message..."></textarea>
+            <button id="submit-review-button" type="submit">Submit</button>
+            <button id="submit-button" type="submit">Submit</button>
+          </form>
+        <script>
+          loadComments();
```

Comment:
This can't be called before the "script.js" is loaded on line 55 below. You can move this below that line, though.